public/locales/ru/common.json

```diff
@@ -726,5 +732,35 @@
     "copy-link": "Копировать ссылку",
     "copy-text": "Копировать текст",
     "disable-share": "Отменить"
+  },
+  "personal": {
+    "subsNewUnits": "новых эпизодов",
+    "unsubscribe": "Вы подписаны",
```

Comment:
Are you sure?

Reply:
button with action unsubscribe where wright that you subscribed